src/MongoDB.Driver.Core/Core/Authentication/SaslMapParser.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections.Generic;
```

Comment:
Used to be be an inner class called NVParser: now shared between the SCRAM-SHA-1/256 authenticators.